addon/-private/utils/send-action.js

```diff
@@ -0,0 +1,11 @@
+import { SUPPORTS_CLOSURE_ACTIONS } from 'ember-compatibility-helpers';
+
+export default function(context, action) {
+  let params = Array.from(arguments).slice(2);
```

Comment:
Why not simply `export default function(context, action, ...params) {`?

Reply:
Lol, just want sure with the syntax or what search terms I would use. So went with the techniques I remembered.